lib/ancestry/instance_methods.rb

```diff
@@ -151,14 +151,30 @@ def ancestor_of?(node)
     # assuming that parent hasn't changed
     def parent= parent
       self.ancestor_ids = parent ? parent.path_ids : []
+      clear_memoized_instance_variables
```

Comment:
think you want to do this work over in ancestry_ids=

Reply:
I'm not sure I believe you.